### ISSUE TITLE
[FR] Native Apple Silicon Support

### ISSUE THREAD

**Issue by Jae0143** [AppFlowy-IO/AppFlowy#1405]
### Description

Native Apple Silicon support missing 

### Impact

By running natively on Apple Silicon, it would increase energy efficiency and responsiveness.

**Comment by annieappflowy** [comment]
GitHub CI doesn't support it yet

**Comment by johnblommers** [comment]
Huh! I did not know this. Shame on GitHub. Apple Silicon is in year two now.

**Comment by rileyhawk1417** [comment]
Correct me if am wrong but in the current AppFlowy releases "macos-universal' is the one which supports the Silicon Chip?

**Comment by johnblommers** [comment]
> Correct me if am wrong but in the current AppFlowy releases "macos-universal' is the one which supports the Silicon Chip?

Yes indeed, I launched it on my M1 Mac Mini and it shows as "Apple" in the Activity Monitor.

**Comment by rileyhawk1417** [comment]
If its working on your hardware then that's great closing this issue as completed.
